skops/saving_utils/default_template.md

```diff
@@ -0,0 +1,27 @@
+"""
+---
+{{ card_data }}
+---
+
+# Model description
+
+This model is part of `super_cool_models` package (which doesn't exist)! It is a fine tuned `cool-model` on the `{{ dataset_name }}`.
+
+## Intended uses & limitations
+
+This model doesn't exist, so you probably don't want to use it! This is just an example template. Please write a very thoughtful model card ❤️
```

Comment:
make sure your lines are not too long. If you're using VSCode, the Rewrap extension is your friend :)